src/main/java/org/aerogear/digger/client/services/JobService.java

```diff
@@ -43,6 +42,12 @@
 
     private final Logger LOG = LoggerFactory.getLogger(JobService.class);
 
+    private boolean crumbFlag = false;
+
+    public JobService(boolean crumbFlag) {
```

Comment:
@matzew  see https://wiki.jenkins-ci.org/display/JENKINS/Remote+access+API and search for "CSRF Protection"